plugin/src/main/resources/org/wildfly/plugins/bootablejar/maven/cloud/openshift-https-script.cli

```diff
@@ -0,0 +1,10 @@
+# HTTPS changes
+
+if (outcome == success) of /core-service=management/security-realm=ApplicationRealm/server-identity=ssl:read-resource
```

Comment:
I assume these are executed in a batch? Otherwise possibly they should be the opposite way round.

Reply:
No batch, fixed.